pkgs/os-specific/darwin/dsymutil/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv }:
+
+assert stdenv.isDarwin;
+
+stdenv.mkDerivation {
+  name = "dsymutil";
+  src = "/usr/bin/dsymutil";
+
+  unpackPhase = "true";
+  dontBuild = true;
```

Comment:
I prefer `phases = [ "installPhase" ];` for things like this, since none of the other phases are needed.